emperor/support_files/js/color-view-controller.js

```diff
@@ -60,29 +60,33 @@ function ColorViewController(container, decompViewDict){
 
   // Build the options dictionary
   var options = {'valueUpdatedCallback':function(e, args) {
-                   var val = args.item.category, color = args.item.color, group = [];
+                   var val = args.item.category, color = args.item.value, group = [];
                    group = args.item.plottables;
-                   scope.decompViewDict[scope.getActiveDecompViewKey()].setGroupColor(color, group);
+                   ColorViewController.setPlottableAttributes(scope.decompViewDict[scope.getActiveDecompViewKey()], color, group);
```

Comment:
Can you move `scope.decompViewDict...` to a separate variable so this line is shorter? We try to keep lines as close to 80 columns as possible, but I know sometimes we can't really do this.